File: Bronze/2455/2455.java
// [2455] 지능형 기차
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException
    {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int max = 0, num = 0;
        String[] s;

        for (int i = 0; i < 4; i++)
        {
            s = br.readLine().split(" ");
            num -= Integer.parseInt(s[0]);  // 내린 사람
            num += Integer.parseInt(s[1]);  // 탄 사람
            if (max < num) max = num;
        }
        System.out.println(max);
    }
}

File: Bronze/2455/2455.cpp
// [2455] 지능형 기차
#include <iostream>
using namespace std;

int main(void) {
    int max = 0, num = 0;
    
    for (int i = 0; i < 4; i++) {
        int in, out;
        cin >> out >> in;
        num -= out;
        num += in;
        if (max < num) max = num;
    }
    cout << max << "\n";
}
